deploy/template.yaml

```diff
@@ -26,6 +26,38 @@ Resources:
           Name: "AWS::Include"
           Parameters:
             Location: "../openAPI/core-back-internal.yaml"
+      AccessLogSetting:
+        DestinationArn: !GetAtt IPVCoreInternalAPILogGroup.Arn
+        Format: >-
+          {
+          "requestId":      "$context.requestId",
+          "ip":             "$context.identity.sourceIp",
+          "requestTime":    "$context.requestTime",
+          "httpMethod":     "$context.httpMethod",
+          "path":           "$context.path",
+          "routeKey":       "$context.routeKey",
+          "status":         "$context.status",
+          "protocol":       "$context.protocol",
+          "responseLatency":"$context.responseLatency",
+          "responseLength": "$context.responseLength"
+          }
+
+  IPVCoreInternalAPILogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub /aws/apigateway/${AWS::StackName}-CoreBackInternal-API-GW-AccessLogs
+      RetentionInDays: 14
+
+  IPVCoreInternalAPILogGroupSubscriptionFilter:
+    Type: AWS::Logs::SubscriptionFilter
+    Properties:
+      DestinationArn: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prod"
+      FilterPattern: ""
+      LogGroupName: !Select # Select the string at the 6th index of...
```

Comment:
You should be able to replace this with simply `!Ref IPVCoreInternalAPILogGroup`